src/CmfRoutingAutoBundle.php

```diff
@@ -27,6 +28,11 @@ public function build(ContainerBuilder $container)
         $this->buildPhpcrCompilerPass($container);
     }
 
+    public function registerCommands(Application $application)
+    {
+        // Do not allow discover automatically commands
```

Comment:
"disable automatic discovery of commands" ?

Reply:
Thanks.